codecov_cli/helpers/ci_adapters/cloudbuild.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from codecov_cli.helpers.ci_adapters.base import CIAdapterBase
+
+
+class GoogleCloudBuildAdapter(CIAdapterBase):
+    # Google Cloud Build uses variable substitutions in the builds
+    # https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values
+    # For these to be available as environment variables, so this adapter
+    # can read the values, you have to manually map the substitution variables to
+    # environment variables on the build step, as documented here:
+    # https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values#map_substitutions_manually
```

Comment:
I was expecting which values needs mapping to be documented here

Reply:
Better now?